### DISCUSSION TITLE
MQTT Status

### DISCUSSION THREAD

**Discussion by dabbelju007** [goecharger/go-eCharger-API-v2#129] [Q&A]
Hallo zusammen,

ich habe eine Frage zum Thema MQTT. Wir haben mehrer goe im Einsatz. Bei einem habe ich das Problem, dass über MQTT nicht alle 5 Sekunden die MQTT Nachricht Status kommt. Stattdessen kommen die Keys einzeln. Hat jemand eine Idee woran das liegt? Auf dem goe Läuft im Moment die FW 0.55.5 Beta, weil wir mit dem Gerät auch das Überschussladen testen.

Danke und viele Grüße
Christian

**Comment by 0xFEEDC0DE64** [comment]
eigentlich sollte doch jede sekunde zumindest die utc zeit sich ändern und damit gepublished werden. gerne mit der seriennummer an den support wenden, wahrscheinlich müssen wir da was untersuchen.
